chainercv/visualizations/vis_point.py

```diff
@@ -6,34 +6,33 @@
 from chainercv.visualizations.vis_image import vis_image
 
 
-def vis_point(img, point, mask=None, ax=None):
+def vis_point(img, point, visible=None, ax=None):
     """Visualize points in an image.
 
     Example:
 
         >>> import chainercv
         >>> import matplotlib.pyplot as plt
-        >>> dataset = chainercv.datasets.CUBPointDataset()
-        >>> img, point, mask = dataset[0]
-        >>> chainercv.visualizations.vis_point(img, point, mask)
+        >>> dataset = chainercv.datasets.CUBKeypointDataset()
+        >>> img, point, visible = dataset[0]
+        >>> chainercv.visualizations.vis_point(img, point, visible)
         >>> plt.show()
 
     Args:
-        img (~numpy.ndarray): An image of shape :math:`(3, height, width)`.
-            This is in RGB format and the range of its value is
-            :math:`[0, 255]`. This should be visualizable using
-            :obj:`matplotlib.pyplot.imshow(img)`
-        point (~numpy.ndarray): An array of point coordinates whose shape is
-            :math:`(P, 2)`, where :math:`P` is
-            the number of points.
-            The second axis corresponds to :math:`y` and :math:`x` coordinates
-            of the points.
-        mask (~numpy.ndarray): A boolean array whose shape is
-            :math:`(P,)`. If :math:`i` th element is :obj:`True`, the
-            :math:`i` th point is not displayed. If not specified,
-            all points in :obj:`point` will be displayed.
+        img (~numpy.ndarray): See the table below.
+            If this is :obj:`None`, no image is displayed.
+        point (~numpy.ndarray): See the table below.
+        visible (~numpy.ndarray): See the table below.
         ax (matplotlib.axes.Axes): If provided, plot on this axis.
 
+    .. csv-table::
+        :header: name, shape, dtype, format
+
+        :obj:`img`, ":math:`(3, H, W)`", :obj:`float32`, \
+        "RGB, :math:`[0, 255]`"
+        :obj:`point`, ":math:`[(K, 2)]`", :obj:`float32`, ":math:`(y, x)`"
```

Comment:
If we assumes all instances have the same `K`, it should be `(N, K, 2)` (`N` is the number of instances).

Reply:
In #808, `R` is used for the number of instances. It's better to make them consistent.

Reply:
We don't assume that all instances have the same `K` for this function.
That is why I left the documentation to accept lists.

Reply:
So, these two lines should be like `(~numpy.ndarray or list of arrays)`.
https://github.com/chainer/chainercv/blob/07f136e1d797ffe8024e95e9b2c74cb2cb602fe2/chainercv/visualizations/vis_point.py#L24-L25